qemu/tests/qemu_guest_agent.py

```diff
@@ -2550,6 +2551,11 @@ def gagent_check_fsinfo(self, test, params, env):
             if params["os_type"] == "windows":
                 mount_pt = mount_pt[:2]
 
+            error_context.context("Check filesystem usage statistics .", logging.info)
+            disk_total_bytes = fs["total-bytes"]
+            disk_used_bytes = fs["used-bytes"]
+            logging.info("The total_bytes of %s is %s,The used_bytes of %s is %s." % (mount_pt, disk_total_bytes, mount_pt, disk_used_bytes))
+
```

Comment:
Check point is missing, you should check if the value is the same with it in guest.

Reply:
oh,right,re-editing.